dart-sdk/rust/src/api/client.rs

```diff
@@ -32,20 +32,31 @@ impl CoreClient {
         base_url: Option<String>,
         #[frb(default = "30000")] poll_interval_ms: u64,
         #[frb(default = "3000")] poll_jitter_ms: u64,
+        #[frb(default = "info")] log_level: String,
     ) -> CoreClient {
-        log::set_max_level(log::LevelFilter::Info);
+        log::set_max_level(log::LevelFilter::Debug);
 
-        let base_url = base_url.unwrap_or_else(|| DEFAULT_BASE_URL.to_owned());
+        {
+            let mut builder = env_logger::Builder::from_env(
+                env_logger::Env::new().write_style("EPPO_LOG_STYLE"),
```

Comment:
Is this a dart/flutter canonical environment style for this kind of parameter? Just asking if a customer sees this will they "get it" or better yet is there a name that will "just work" without them reading a doc?

Reply:
I'm not sure. I took this partially from the ruby-sdk client https://github.com/Eppo-exp/eppo-multiplatform/blob/main/ruby-sdk/ext/eppo_client/src/client.rs#L77-L81
however, I haven't figured out a way to pass environment variables from flutter all the way down into the rust native code, not sure if it's even possible, so we may want to remove this `EPPO_LOG_STYLE` - as usual, I'l let @rasendubi chime in